--- a/Projects/UOContent/Misc/AOS.cs	2026-09-06 11:54:37.250685700 -0500
+++ b/Projects/UOContent/Misc/AOS.cs	2026-09-06 11:54:37.884756700 -0500
@@ -580,6 +580,15 @@
                         value += attrs[attribute];
                     }
                 }
+                else if (obj is PetMimic mimic)
+                {
+                    var attrs = mimic.Attributes;
+
+                    if (attrs != null)
+                    {
+                        value += attrs[attribute];
+                    }
+                }
             }
 
             return value;
