--- "a/Projects/UOContent/Items/Skill Items/Misc/Bandage.cs"	2026-09-06 11:54:37.254189800 -0500
+++ "b/Projects/UOContent/Items/Skill Items/Misc/Bandage.cs"	2026-09-06 11:54:37.885761700 -0500
@@ -118,6 +118,12 @@
                 return;
             }
 
+            if (targeted is PetMimic mimic)
+            {
+                mimic.TryFeed(from, _bandage);
+                return;
+            }
+
             from.SendLocalizedMessage(500970); // Bandages can not be used on that.
         }
 
@@ -130,6 +136,10 @@
                     _bandage.Consume();
                 }
             }
+            else if (targeted is PetMimic mimic)
+            {
+                mimic.TryFeed(from, _bandage);
+            }
             else
             {
                 base.OnNonlocalTarget(from, targeted);
