--- a/Projects/UOContent/Engines/Craft/Core/CraftGump.cs	2026-09-06 11:54:37.261199800 -0500
+++ b/Projects/UOContent/Engines/Craft/Core/CraftGump.cs	2026-09-06 11:54:37.887265500 -0500
@@ -444,6 +444,11 @@
 
             if (context != null)
             {
+                // ClusterF: Make X - cancel any in-flight repeat for a manual single craft
+                context.RepeatCount   = 0;
+                context.RepeatItem    = null;
+                context.RepeatTypeRes = null;
+
                 var res = item.UseSubRes2 ? _craftSystem.CraftSubRes2 : _craftSystem.CraftSubRes;
                 var resIndex = item.UseSubRes2 ? context.LastResourceIndex2 : context.LastResourceIndex;
 
