--- a/Projects/UOContent/Misc/Loot.cs	2026-09-29 14:52:36.010122900 -0500
+++ b/Projects/UOContent/Misc/Loot.cs	2026-09-29 14:52:59.022856300 -0500
@@ -315,7 +315,8 @@
             typeof(SkullCap), typeof(Bandana), typeof(FloppyHat),
             typeof(Cap), typeof(WideBrimHat), typeof(StrawHat),
             typeof(TallStrawHat), typeof(WizardsHat), typeof(Bonnet),
-            typeof(FeatheredHat), typeof(TricorneHat), typeof(JesterHat)
+            typeof(FeatheredHat), typeof(TricorneHat), typeof(JesterHat),
+            typeof(OrcMask), typeof(TribalMask)
         };
 
         public static Type[] LibraryBookTypes { get; } =
